DSharpPlus.Interactivity/InteractivityExtension.cs

```diff
@@ -625,6 +625,39 @@ public async Task<ReadOnlyCollection<Reaction>> CollectReactionsAsync(DiscordMes
             await this._compPaginator.DoPaginationAsync(req).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Sends a paginated message in response to an interaction.
+        /// <para>
+        /// <b>Pass the interaction directly. Interactivity will ACK it.</b>
+        /// </para>
+        /// </summary>
+        /// <param name="interaction">The interaction to create a response to.</param>
+        /// <param name="ephemeral">Whether the response should be ephermaral.</param>
+        /// <param name="user">The user to listen for button presses from.</param>
+        /// <param name="pages">The pages to paginate.</param>
+        /// <param name="buttons">Optional: custom buttons</param>
+        /// <param name="behaviour">Pagination behaviour.</param>
+        /// <param name="deletion">Deletion behaviour</param>
+        /// <param name="token">A custom cancellation token that can be cancelled at any point.</param>
+        public async Task SendPaginatedMessageAsync(DiscordInteraction interaction, bool ephemeral, DiscordUser user, IEnumerable<Page> pages, PaginationButtons? buttons = null, PaginationBehaviour? behaviour = default, ButtonPaginationBehavior? deletion = default, CancellationToken token = default)
+        {
+            var bhv = behaviour ?? this.Config.PaginationBehaviour;
+            var del = deletion ?? this.Config.ButtonBehavior;
+            var bts = buttons ?? this.Config.PaginationButtons;
+
+            var builder = new DiscordInteractionResponseBuilder()
+                .WithContent(pages.First().Content)
+                .AddEmbed(pages.First().Embed)
+                .AsEphemeral(ephemeral);
+
+            await interaction.CreateResponseAsync(InteractionResponseType.ChannelMessageWithSource, builder);
```

Comment:
throws 404 if we create a DeferredChannelMessageWithSource response before calling this

*sorry*

Reply:
It's very clearly stated in the documentation.

Reply:
sorry, didnt see it, but shouldnt there be a way to *not* ack it?

Reply:
What sense does that make

Reply:
The ability to make it ack requires making this method far clunkier